bpf/runtime.h

```diff
@@ -0,0 +1,30 @@
+#ifndef RUNTIME_SUPPORT_H
+#define RUNTIME_SUPPORT_H
+
+#include "vmlinux.h"
+#include "bpf_helpers.h"
+#include "bpf_builtins.h"
+#include "pid_types.h"
+#include "nodejs.h"
+
+static __always_inline u64 extra_runtime_id() {
```

Comment:
Meant to support other runtimes which have internal threading models, for now only checks NodeJS.